UPGRADING.rst

```diff
@@ -25,8 +25,8 @@ to ::
     'AZUL_SUBDOMAIN_TEMPLATE': '*.{AZUL_DEPLOYMENT_STAGE}',
 
 
-#1272 Use Lambda layers to speed up ``make deploy``
```

Comment:
Revert the changes to this file. These instructions should match source at the commit that introduced them. We shouldn't retroactively change them.